.github/workflows/psalm-security-analysis.yml

```diff
@@ -14,7 +14,7 @@ jobs:
         uses: docker://vimeo/psalm-github-actions
         with:
           composer_require_dev: true
-          composer_ignore_platform_reqs: false # https://github.com/psalm/psalm-github-actions/pull/29
+          composer_ignore_platform_reqs: true
```

Comment:
Note that this got merged a couple of days ago, so undoing the existing workaround.